app/assets/javascripts/discourse/app/components/sidebar/more-section-link.hbs

```diff
@@ -1,15 +1,37 @@
-<Sidebar::SectionLink
-  @shouldDisplay={{@sectionLink.shouldDisplay}}
-  @linkName={{@sectionLink.name}}
-  @route={{@sectionLink.route}}
-  @href={{@sectionLink.href}}
-  @query={{@sectionLink.query}}
-  @title={{@sectionLink.title}}
-  @content={{@sectionLink.text}}
-  @currentWhen={{@sectionLink.currentWhen}}
-  @badgeText={{@sectionLink.badgeText}}
-  @model={{@sectionLink.model}}
-  @models={{@sectionLink.models}}
-  @prefixType={{@sectionLink.prefixType}}
-  @prefixValue={{@sectionLink.prefixValue}}
-/>
```

Comment:
this is Added so more section is handling external links and buttons in case admin adds them in more segment